components/FeatureFeed/FeatureFeed.js

```diff
@@ -12,10 +12,22 @@ import { getComponent } from 'utils';
 
 // This component is created to map the features by type and send them.
 const FeatureFeed = (props = {}) => {
-  const CardListComponents = {
+  const FEATURE_COMPONENTS = {
```

Comment:
I'm not 100% sure this concern is well founded, but this feels like it should be statically defined outside a `render` method to avoid issues.

Could we move this up and out of the function definition?